controllers/flowcollector_controller_test.go

```diff
@@ -630,7 +630,7 @@ func GetReadyCR(key types.NamespacedName) *flowsv1alpha1.FlowCollector {
 			return err
 		}
 		cond := meta.FindStatusCondition(cr.Status.Conditions, conditions.TypeReady)
-		if cond.Status == metav1.ConditionFalse {
+		if cond != nil && cond.Status == metav1.ConditionFalse {
```

Comment:
@jpinsonneau maybe the problem is here: if cond is nil, then it should return an error ?

Reply:
thanks it seems to work as expected :+1: